### ISSUE TITLE
[Documentation] MDC Components do not automatically respond to Increased Contrast accessibility feature

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#4339]
## Description
MDC component do not automatically adjust their colors for "increased contrast"  ([UIAccessibilityDarkerSystemColorsEnabled()](https://developer.apple.com/documentation/uikit/1615087-uiaccessibilitydarkersystemcolor)). iOS will automatically adjust `tintColor` values to be darker than the set value, but other colors (like surface colors or title colors) are not. 

The general effect may lead to decreased contrast in MDC-built apps when the actual desire is increased contrast for the user.  Apple's designs tend to build from white, so darkening tint/chrome is usually the correct approach. However, since many of our components use colored surfaces (like AppBar), this can result in decreased contrast ratios.

## Acceptance Criteria
1. Our documentation, either per-component or as a system, describes how to work with UIAccessibilityDarkerSystemColorsEnabled() to make appropriate color adjustments to our components.
1. The [Color Schemes component](https://github.com/material-components/material-components-ios/tree/develop/components/schemes/Color) provides a high-contrast semantic color scheme based on the current Material defaults.
1. The MDC Catalog app responds automatically to UIAccessibilityDarkerSystemColorsEnabled() by switching to a high-contrast color scheme that achieves [WCAG 2.0 Level AAA](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) for color contrast as much as possible.

---
#### Internal data
- Associated internal bug: [b/117178980](http://b/117178980)
- Blocked by: https://github.com/material-components/material-components-ios/issues/7522

**Comment by romoore** [comment]
Needs a designer to sign-off on an engineer's proposed high-contrast color scheme.

**Comment by romoore** [comment]
This is not required for the accessibility leveling project.

**Comment by romoore** [comment]
Closing this due to inactivity greater than 6 months. Please feel free to reopen if you believe it should be worked on in the near future.

**Comment by romoore** [comment]
Since the dupe was more recently filed.
